packages/venia-ui/lib/util/swUtils.js

```diff
@@ -1,3 +1,7 @@
+export const SHOULD_ENABLE_SERVICE_WORKER =
```

Comment:
This sounds like a const that would be used to enable the service worker.

Can you name it, `SERVICE_WORKER_ENABLED`? Would be more fitting, IMO.

Reply:
If `true` it does not mean the SW is enabled yet. It just is a condition to enable or not. `SERVICE_WORKER_ENABLED` sounds like a boolean to say if the SW is enabled or not.

Reply:
I'm saying if the const is being used in places as a way of saying "Is the service worker enabled? If so, do X" then the proper name should be `SERVICE_WORKER_ENABLED`.

When you say "enabled yet" I think you mean that it hasn't been created or something. It's a difference of semantics and I'm not super worried about it. Just letting you know how it reads to me.

Reply:
I was actually going to post a similar comment. In `registerSW`, `SHOULD_ENABLE_SERVICE_WORKER` reads fine because it's job is to then enable the service worker (if `true`).

In other places (such as `useProductImageCarousel` in this PR), it reads funny because it's doing something completely different than enabling the service worker when `SHOULD_ENABLE_SERVICE_WORKER` is `true`.

Unfortunately I can't really think of anything that reads well in both scenarios. In that case I say stick with the convention of `DEV_SERVER_SERVICE_WORKER_ENABLED` and just call it `SERVICE_WORKER_ENABLED` with the understanding that `registerSW` actually has to enable it first.

Reply:
How about `VALID_SERVICE_WORKER_ENVIRONMENT`?